.doc_gen/metadata/kms_metadata.yaml

```diff
@@ -684,16 +754,27 @@ kms_Scenario_KeyEncryption:
   services:
     kms: {Encrypt, Decrypt, ReEncrypt}
 kms_Scenario_KeyManagement:
-  title: Manage &kms-keys-long; using an &AWS; SDK
-  title_abbrev: Manage keys
+  title: Learn core operations for &kms-key; using an &AWS; SDK
+  title_abbrev: Learn &kms-key; core operations
   synopsis_list:
     - Create a &kms-key;.
     - List &kms-keys; for your account and get details about them.
     - Enable and disable &kms-keys;.
     - Generate a symmetric data key that can be used for client-side encryption.
+    - Generate an asymmetric key used to digitally sign data.

```

Comment:
I think these should be 2 scenarios.
The python scenario currently does not demonstrate tagging or asymmetric keys, so the description is incorrect.

Reply:
Once Pyhthon follow is updated - it will.